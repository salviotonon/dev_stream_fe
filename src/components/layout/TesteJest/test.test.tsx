import { screen, render } from '@testing-library/react';

import { TesteJest } from '.';

describe('<TesteJest />', () => {
  it('should render the heading', () => {
    const { container } = render(<TesteJest />);

    expect(screen.getByRole('heading', { name: /react/i })
    ).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
