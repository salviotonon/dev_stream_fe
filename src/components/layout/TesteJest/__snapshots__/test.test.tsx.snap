// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TesteJest /> should render the heading 1`] = `
<main>
  <h2>
    React
  </h2>
</main>
`;
